builtin/providers/aws/cloudfront_distribution_configuration_structure.go

```diff
@@ -124,12 +124,16 @@ func flattenDistributionConfig(d *schema.ResourceData, distributionConfig *cloud
 			return err
 		}
 	}
+
 	if distributionConfig.Logging != nil && *distributionConfig.Logging.Enabled {
 		err = d.Set("logging_config", flattenLoggingConfig(distributionConfig.Logging))
-		if err != nil {
-			return err
-		}
+	} else {
+		err = d.Set("logging_config", []interface{}{})
```

Comment:
Hey @jrnt30, I think right here, you'd want to do the following instead:

```
err = d.Set("logging_config", schema.NewSet(loggingConfigHash, []interface{}{}))
```

The idea here being that even though it's an empty set you still probably want to pass a `*schema.Set` and the correct hash function.